### PR TITLE
Bug 1229084 - Organize device-specific APIs under /devices/:id

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

File: src/routes/devices.js [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+
+    newSession.id = sessionId;
+    return sessions[sessionId] = newSession;
+  });
+
+  res.json(deviceSessions);
+}
+
+async function getOne(req, res) {
+  let adb = req.adb;
```
Comment (gaye) [inline src/routes/devices.js]: nit: `let {adb} = req` for maximum esnext points?

Comment (eliperelman) [inline src/routes/devices.js]: Good catch!

File: src/routes/devices.js [hunk 2]
```diff
@@ -1,14 +1,55 @@
...
+    fxos.readGeckoCommit(adb)
+  ]);
+
+  let result = Object.assign(info, {gaia: gaiaCommit, gecko: geckoCommit});
+  res.json(result);
+}
+
+router.get('/', getAll);
+router.get('/:id', getOne);
+router.use('/:id/connections', require('./connections'));
```
Comment (gaye) [inline src/routes/devices.js]: This makes me really happy you did all of the endpoints stuff before

Comment (eliperelman) [inline src/routes/devices.js]: Hehe, agreed.

**Pull request merged**